modules/va_notify/app/controllers/va_notify/callbacks_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module VANotify
+  class CallbacksController < VANotify::ApplicationController
+    include ActionController::HttpAuthentication::Token::ControllerMethods
+
+    service_tag 'va-notify'
+
+    skip_before_action :verify_authenticity_token, only: [:create]
+    skip_before_action :authenticate, only: [:create]
+
+    before_action :authenticate_callback
+
+    def create
+      Rails.logger.info { "Notification received: #{params.inspect}" }
+      notification_id = params[:id]
+
+      if (notification = VANotify::Notification.find_by(notification_id: notification_id))
+        Rails.logger.info("va_notify callbacks - Updating notification #{notification.id}")
+        notification.update(notification_params)
+      else
+        Rails.logger.info("va_notify callbacks - Received update for unknown notification #{notification_id}")
+      end
+
+      render json: { message: 'success' }, status: :ok
+    end
+
```

Comment:
everything below the `create` action would benefit from being `private` if it's only being called from within the controller

Reply:
Fixed